### PR TITLE
Update: Extend InstalledApp to configure app and show button

### PR DESCRIPTION
**Pull request by ericrpowers**
#### Description
This PR extends the installed app model to house whether a highwind app will need additional configuration on startup and if it will show a button in the UI.

### CODE REVIEW

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "58a74f112c4aaf78a7ef341dd817ea56d84b451b",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "c3bdace255b33302d16da10199feb328fcb3aea4",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "d782d7b320b8dd05e8d3fab6b5daf29e2362bff7",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "concourse-pipelines",
    "pr-id": "697",
    "commit-sha": "c60487ea3ffd4c40289b9b3202de7128510695da",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "58a74f112c4aaf78a7ef341dd817ea56d84b451b",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "c3bdace255b33302d16da10199feb328fcb3aea4",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "d782d7b320b8dd05e8d3fab6b5daf29e2362bff7",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "b82786ce9db8672d0c298df4885f5808a5a3a47c",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "c3bdace255b33302d16da10199feb328fcb3aea4",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "d782d7b320b8dd05e8d3fab6b5daf29e2362bff7",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "b82786ce9db8672d0c298df4885f5808a5a3a47c",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "c3bdace255b33302d16da10199feb328fcb3aea4",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "d782d7b320b8dd05e8d3fab6b5daf29e2362bff7",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "b82786ce9db8672d0c298df4885f5808a5a3a47c",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "f7a2f65bc86d36acc1bdd6f462537aef26ede17d",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "d782d7b320b8dd05e8d3fab6b5daf29e2362bff7",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "4bc39b65d87d5bbd103a341fafbb822c51506b6e",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "d782d7b320b8dd05e8d3fab6b5daf29e2362bff7",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "02e53e98e025bcb491747584105f645e61265464",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "7f73216a8c3cffdd9edb17d0e8871cd6df2f6ddd",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "5f59f98eced38642eedc36bc02b0c6ed0a991cca",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "643f14e64d8b51a326d3c432970a4ebd538ef9c0",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "835b65488f3c03caf992f67097c5499b1bb41951",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "275807e14b2be46fbcb4966157c61ab4175e8093",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "643f14e64d8b51a326d3c432970a4ebd538ef9c0",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "1ed7bfa434cc0060a78ec418d2ff7a0e3f40b24d",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "cb4ada455bbb91570028b98017ae66f3c11fe673",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "f88096251d4933c7249b43e1d68bac780b586387",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "cb4ada455bbb91570028b98017ae66f3c11fe673",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "fb28701a3a9b2a6470c8a99a4536d93858cae0b9",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "ab8020211a12fd64c127f90bab17814bce5a068c",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "29217a9080b861004572a9a3d998a6a1523854b5",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "a031aa0598d366dcd75c11c5573f7925414b93e1",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "clad",
    "pr-id": "1532",
    "commit-sha": "71e4720d2603db984effbe6f59e8ae72eff08ce5",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaiajs",
    "pr-id": "30",
    "commit-sha": "81290da602788555e1ef7643096717553089eae0",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "concourse-pipelines",
    "pr-id": "697",
    "commit-sha": "739bcf06b45ac47d7de0f3ac25512840c9c634bd",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "01ae89344441ec4135ed3f1f2144174e59a0f6ce",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "twistlock",
    "pr-id": "2",
    "commit-sha": "9712f993f268c45444d2796c488cdcf57cea0886",
    "pipeline": "master"
  }
]
```

File: .spelling [hunk 1]
```diff
@@ -276,6 +278,7 @@ ValidateUIParameter
 vCPUs
 vince
 VMs
+vulscan
```
Comment (t00f) [inline .spelling]: This line can be removed

File: specs/complianceissue.spec [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Model
+model:
+  rest_name: complianceissue
+  resource_name: complianceissues
+  entity_name: ComplianceIssue
+  package: aki
+  group: integration/vulscan
```
Comment (t00f) [inline specs/complianceissue.spec]: should be "aki/vulnerabilities"

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "integ-step1",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "7376018f307bf0d7d829797138a060517074ef81",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "698807578258fbfcdb36fc76856bda4eadd22904",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "clad",
    "pr-id": "1532",
    "commit-sha": "c184213df3628ba7c243b4eadb66f0393f35917a",
    "pipeline": "master"
  },
  {
    "project": "integ-step1",
    "component": "gaiajs",
    "pr-id": "30",
    "commit-sha": "81290da602788555e1ef7643096717553089eae0",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "7376018f307bf0d7d829797138a060517074ef81",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "698807578258fbfcdb36fc76856bda4eadd22904",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "661",
    "commit-sha": "d51353272a77905d95a5148e9d1f695df0462d45",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "446",
    "commit-sha": "698807578258fbfcdb36fc76856bda4eadd22904",
    "pipeline": "master"
  }
]
```

**Pull request merged**